tools/DocsLinter/Tests/SimplifiedMarkdownDocTests.cs

```diff
@@ -8,33 +8,6 @@ namespace DocsLinter.Tests
     [TestFixture]
     public class SimplifiedMarkdownDocTests
     {
-        [Test]
-        public void ParseLink_should_return_RemoteLink_when_url_starts_with_http()
-        {
-            var linkInline = new LinkInline("https://www.google.com", "Title");
-            var link = SimplifiedMarkdownDoc.ParseLink(linkInline);
-
-            Assert.IsInstanceOf<RemoteLink>(link);
-        }
-
-        [Test]
-        public void ParseLink_should_return_RemoteLink_when_url_starts_with_www()
-        {
-            var linkInline = new LinkInline("www.google.com", "Title");
```

Comment:
These tests are probably still valuable right? To ensure we catch various types of links

Reply:
check out the new tests